### DISCUSSION TITLE
Don't know what to put for BASE_URL

### DISCUSSION THREAD

**Discussion by shadowaero** [mealie-recipes/mealie#3874] [General]
I can't seem to figure out what to enter for BASE_URL environment Value.  I've tried my public IP.  URL for mealie.website.com  I've tried localhost and 127.0.0.1:8080.  Can anyone suggest what should be there?  I also have same issue with Host.

**Comment by Kuchenpirat** [comment]
You enter the adress mealie is reachable under. 
E.G. if you own the domain example.com and mealie is hosted at the subdomain mealie.example.com you would input `mealie.example.com` into the env. 

It should be noted that the BASE_URL env is not strictly needed to run mealie. It is mostly used to create invite and password reset links. So for a first try it should not matter much what you put there.
